src/lib.rs

```diff
@@ -25,6 +25,42 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+//! In 1975, a small team of engineers led by [Chuck Peddle] at MOS Technology
```

Comment:
I think there are two audiences here: People wanting to read the documentation of the code, and people wanting to read about the 6502. Both are worth writing for, but I wonder if this history wouldn't be better off in a separate document, like `README.md`, or even a new `HISTORY.md`.

Reply:
very true. I've initially oved it to `HISTORY.md`, but later moved it into the `README.md` as it seemed to be a better fit because it wasn't too long and made for a nice "ending" of the docs.